src/main/io/serial_msp.c

```diff
@@ -750,6 +750,13 @@ static bool processOutCommand(uint8_t cmdMSP)
         serialize16(constrain(averageSystemLoadPercent, 0, 100));
         break;
 
+    case MSP_NAME:
+        headSerialReply(MAX_NAME_LENGTH);
```

Comment:
Yes good comment, I actually started doing that but ran into some problem with the compiler didn't allow declaration of variables within labels and then I ran out of time yesterday to fix it. Will look further into this later tonight.